openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -0,0 +1,533 @@
+
+    def __init__(self,
+                 stateA: ChemicalState,
+                 stateB: ChemicalState,
+                 ligandmapping: LigandAtomMapping,
+                 settings: Union[Dict, RelativeLigandTransformSettings] = None,
+                 ):
+        """
+        Parameters
+        ----------
+        stateA, stateB : ChemicalSystem
+          the two ligand SmallMoleculeComponents to transform between.  The
+          transformation will go from ligandA to ligandB.
+        ligandmapping : LigandAtomMapping
+          the mapping of atoms between the two ligand components
+        settings : RelativeLigandTransformSettings
+          the settings for the Method.
+
+        The default settings for this method can be accessed via the
+        get_default_settings method,
+        """
+        self._stateA = stateA
+        self._stateB = stateB
+        self._mapping = ligandmapping
+        self._settings = self.__class__.get_default_settings()
+        # TODO: Figure out how to supply partial settings?
+        #if settings is not None:
+        #    self._settings.update(settings)
+
+        # Checks on the inputs!
+        # check that both states have solvent and ligand
+        for state, label in [(stateA, 'A'), (stateB, 'B')]:
+            if 'solvent' not in state.components:
+                raise ValueError(f"Missing solvent in state {label}")
+            if 'ligand' not in state.components:
+                raise ValueError(f"Missing ligand in state {label}")
+        # check that both states have same solvent
+        if not stateA.components['solvent'] == stateB.components['solvent']:
+            raise ValueError("Solvents aren't identical between states")
+        # check that the mapping refers to the two ligand components
+        if stateA.components['ligand'] != ligandmapping.mol1:
+            raise ValueError("Ligand in state A doesn't match mapping")
+        if stateB.components['ligand'] != ligandmapping.mol2:
+            raise ValueError("Ligand in state B doesn't match mapping")
+
+    @classmethod
+    def get_default_settings(cls) -> RelativeLigandTransformSettings:
+        """A dictionary of initial settings for this creating this Protocol
+
+        These settings are intended as a suitable starting point for creating
+        an instance of this protocol.  It is recommended, however that care is
+        taken to inspect and customize these before performing a Protocol.
+
+        Returns
+        -------
+        RelativeLigandTransformSettings
+          a set of default settings
+        """
+        return RelativeLigandTransformSettings(
+            system_settings=SystemSettings(
+                constraints='HBonds'
+            ),
+            topology_settings=TopologySettings(
+                forcefield={'protein': 'amber99sb.xml',
+                            'ligand': 'openff-2.0.0.offxml',
+                            'solvent': 'tip3p.xml'},
```

Comment:
Don't accept this as a change yet, I need to check it works (was getting issues with 14sb originally, hence why I dropped it from the notebook).

```suggestion
                forcefield={'protein': 'amber14-all.xml',
                            'ligand': 'openff-2.0.0.offxml',
                            'solvent': 'amber14/tip3p.xml'},
```